python/rmm/tests/test_rmm.py

```diff
@@ -586,6 +586,34 @@ def test_cuda_async_memory_resource_threshold(nelem, alloc):
     array_tester("u1", 2 * nelem, alloc)  # should trigger release
 
 
+@pytest.mark.parametrize(
+    "mr", [rmm.mr.CudaMemoryResource, rmm.mr.CudaAsyncMemoryResource]
+)
+def test_limiting_resource_adaptor(mr):
+    if not _CUDAMALLOC_ASYNC_SUPPORTED:
+        pytest.skip("cudaMallocAsync not supported")
+
+    cuda_mr = mr()
+
+    allocation_limit = 1 << 20
```

Comment:
Is this an arbitrarily chosen, or special, number? I guess it's such that you know two buffers will be a "nice" number and should therefore fit due to no overallocation from alignment?

Reply:
It's arbitrarily chosen, 1MiB, as you pointed out it's also a "nice" number for which we don't have overallocation due to alignment. Same as above, any strong preferences on what to choose here?

Reply:
I think this is fine as is.